.travis.yml.tmpl

```diff
@@ -12,10 +12,6 @@ arch:
   - amd64
   - arm64
 
-jobs:
```

Comment:
Was this accidentally removed?

Reply:
No, this is by intention. PR #10540 deleted `allow_failures` from `.travis.yml` but not from
`.travis.yml.tmpl`. This leads to `allow_failures` being added again
when running `make update-travis-go-version`. Thus I added commit 034f8a9dde79ffaa93f3f6ef43ec3b5501a5042b to this PR.

Reply:
LGTM. We should have a checker that check if the `template` files are out of sync with the generated files. Not blocking for this PR ofc.